### PR TITLE
cleaning up introduction and removing content that goes off track

### PR DESCRIPTION
**Pull request by vsoch**
This pull request is a first shot at cleaning up the manuscript, namely:

 - the introduction side tracked into talking about research compendiums, which was distracting and out of scope for an introduction that should be focused on leading into talking about building data science containers. 
 - I removed lines that (although possibly true / meaningful) didn't add to the flow of the paper.

I can add comment for any specific choice for reviewers that are interested.

Signed-off-by: vsoch <vsochat@stanford.edu>

### CODE REVIEW

File: ten-simple-rules-dockerfiles.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Ten Simple Rules for Writing Dockerfiles for Reproducible Research"
+title: "Ten Simple Rules for Writing Dockerfiles for Reproducible Data Science"
```
Comment (nuest) [inline ten-simple-rules-dockerfiles.Rmd:2]: Good!

File: ten-simple-rules-dockerfiles.Rmd [hunk 2]
```diff
@@ -61,54 +61,41 @@ Submission notes:
...
 By providing this recipe, authors of scientific articles greatly improve their work's level of documentation, transparency, and reusability.
 Such practice is one important part of common practices for scientific computing [@wilson_best_2014; @wilson_good_2017], with the result that it is much more likely both the author and others are able to reproduce and extend an analysis workflow.
 The containers built from these recipes are portable encapsulated snapshots of a specific computing environment.
 Such containers have been demonstrated for capturing scientific notebooks [@rule_ten_2019] and reproducible workflows [@sandve_ten_2013].
-Research compendia also allow for proper citation of the used computing environment, which is not possible within containers alone.
-Best practices are still a work in progress [cf. @katz_software_2018], but you should try your best to give credit to creators of software you rely on by following recommendations of projects such as CodeMeta ([https://codemeta.github.io/](https://codemeta.github.io/)) and the Citation File Format ([https://citation-file-format.github.io/](https://citation-file-format.github.io/)).
```
Comment (nuest) [inline ten-simple-rules-dockerfiles.Rmd:78]: Killed one of my darlings here... good job :-).

File: ten-simple-rules-dockerfiles.Rmd [hunk 3]
```diff
@@ -369,7 +337,7 @@ You can view all labels with [`docker inspect`](https://docs.docker.com/engine/r
 Labels serve as structured metadata that can be leveraged by services, e.g., https://microbadger.com/labels.
 For example, software versions, license, and maintainer contact information are commonly seen and very useful if a `Dockerfile` is discovered out of context.
 While you can add arbitrarily complex information with labels, for research compendia the user-facing documentation is much more important.
-If you want to earn extra points, and you never know what future algorithms will be able to make sense of, include global identifiers such as [ORCID identifiers](https://orcid.org/) for people, a DOI of the research compendium, e.g., [reserved on Zenodo](https://help.zenodo.org/) before publishing the research compendium, or your funding agency's grant number.
+Important metadata that might be more utilized with future tools includes global identifiers such as [ORCID identifiers](https://orcid.org/), DOIs of the research compendium, e.g., [reserved on Zenodo](https://help.zenodo.org/), or a funding agency's grant number.
```
Comment (nuest) [inline ten-simple-rules-dockerfiles.Rmd]: I've re-added the research compendium link here, but am fine with not using the term throughout the article.

File: ten-simple-rules-dockerfiles.Rmd [hunk 4]
```diff
@@ -638,69 +603,53 @@ This variety of approaches render seemingly more convenient uncontainerised envi
...
-You should include instructions how to import and run the workflow based on the image archive and add your own image tags for clarity.
+You should include instructions for how to import and run the workflow based on the image archive and add your own image tags for clarity.
 Depositing the image next to other project files, i.e., data, code, and the used `Dockerfile`, in a public repository makes them likely to be preserved, but is is highly unlikely that over time you will be able to recreate it precisely from the accompanying `Dockerfile`.
 Publishing the image and the contained metadata therein (e.g., the Docker version used) may even allow future science historians to emulate the Docker runtime environment.
-Applying proper preservation strategies (cf. [@emsley_framework_2018]) can be highly complex, but simply running an image "as-is", i.e. with the default command and entrypoint (see \ruleref{rule:interactive}), and observing the output is quite likely to work for many years into the future.
```
Comment (nuest) [inline ten-simple-rules-dockerfiles.Rmd:699]: Another one of my darlings :skull:  !! I agree though that container preservation as a topic is not mature enough yet to expose the target audience of this article.

**Comment by vsoch** [comment]
Woohoo! Thank you @nuest !

**Pull request merged**